tictactoe/test/console_io_tests.erl

```diff
@@ -1,13 +1,30 @@
 -module(console_io_tests).
 -include_lib("eunit/include/eunit.hrl").
--import(console_io, [display/1,greet_user/0]).
--import(instructions, [greet/0]).
+
+display_test() -> 
```

Comment:
None of these tests here are valuable because you are essentially calling the same methods in the test that the method uses to test equality. Of course they will be equal. 

Figure out an Erlangey way to test this. Figure out what result you are trying to achieve. Maybe you need to rewrite the method to make it easier to test.